test/com/facebook/buck/swift/SwiftDescriptionsTest.java

```diff
@@ -57,13 +57,13 @@ public void testPopulateSwiftLibraryDescriptionArg() throws Exception {
     args.supportedPlatformsRegex = Optional.absent();
 
     SwiftDescriptions.populateSwiftLibraryDescriptionArg(pathResolver, output, args, buildTarget);
-    assertThat(output.moduleName.get(), Matchers.equalTo("bar"));
-    assertThat(output.srcs.get(), Matchers.equalTo(ImmutableSortedSet.<SourcePath>of(swiftSrc)));
+    assertEquals(output.moduleName.get(), "bar");
+    assertEquals(output.srcs.get(), ImmutableSortedSet.<SourcePath>of(swiftSrc));
 
     args.moduleName = Optional.of("baz");
 
     SwiftDescriptions.populateSwiftLibraryDescriptionArg(pathResolver, output, args, buildTarget);
-    assertThat(output.moduleName.get(), Matchers.equalTo("baz"));
```

Comment:
We use `assertThat` as much as possible so that the failure print outs are consistent.